### ISSUE TITLE
Units of `compute_current()` in channels do not match NEURON

### ISSUE THREAD

**Issue by michaeldeistler** [jaxleyverse/jaxley#274]
- Conversion to `uA/cm^2` of external input is [here](https://github.com/jaxleyverse/jaxley/blob/7311cd70e04eb87402041b816ddec498b323ed6d/jaxley/modules/base.py#L742)
- `compute_current()` of channels also returns `mS/cm^2 * mV`, which is `uA/cm^2`, see [here](https://github.com/jaxleyverse/jaxley/blob/7311cd70e04eb87402041b816ddec498b323ed6d/jaxley/channels/hh.py#L51-L60)
- As written [here](https://www.neuron.yale.edu/neuron/static/docs/units/unitchart.html) the capacitance in `um/cm2`.

In summary: all is correct, but (unlike is written in the documentation), the `compute_current()` method of a `Channel` returns the current in `uA/cm2`, **not** in `mA/cm2`.
